crates/utils/src/utils/validation.rs

```diff
@@ -113,7 +58,7 @@ pub fn is_valid_actor_name(name: &str, actor_name_max_length: i32) -> LemmyResul
 fn has_3_permitted_display_chars(name: &str) -> bool {
```

Comment:
Then we dont need this function anymore and can instead check `!name.contains(INVISIBLE_CHARS)`

Reply:
Yep the library includes more invisible chars than we originally had hardcoded. But I believe this does fix #5691, because it now also runs the `has_3_permitted_display_chars` check on post titles, which wasn't done before.

So this PR 

- Adds that check
- Uses a fuller list of invisible chars.